src/App.tsx

```diff
@@ -1,9 +1,29 @@
+import { Navigate, Route, Routes } from 'react-router-dom';
 import './App.scss';
+import { Header } from './Components/Header';
+import { HomePage } from './Components/HomePage';
+import { NoPageFound } from './Components/NoPageFound';
+import { PeopleTable } from './Components/PeopleTable';
 
-const App = () => (
-  <div className="App">
-    <h1>People table</h1>
-  </div>
-);
+const App: React.FC = () => {
+  return (
+    <div className="App">
+      <Header />
+
+      <Routes>
+        <Route
+          path="/"
+          element={<HomePage />}
+        />
+        <Route path="/home" element={<Navigate to="/" />} />
```

Comment:
`replace`?
here is why – https://reach.tech/router/api/navigate
also – this one
![Screen Shot 2022-08-05 at 09 31 42](https://user-images.githubusercontent.com/75496546/183015874-4febb8dc-7972-4e73-a320-65a2a52a4aca.png)